MAINTAINER.md

```diff
@@ -0,0 +1,15 @@
+# Instructions for maintainers of the nodereport project
+
+## Releasing to www.npmjs.com

```

Comment:
Fair enough, "publish to npm" is clear enough. Anybody with the rights to publish nodereport should know what they are publishing to.